scripts/determine-reboot-cause

```diff
@@ -254,7 +254,7 @@ def main():
     previous_reboot_cause, additional_reboot_info = determine_reboot_cause()
 
     # Current time
-    reboot_cause_gen_time = str(datetime.datetime.now().strftime('%Y_%m_%d_%H_%M_%S'))
+    reboot_cause_gen_time = str(datetime.datetime.utcnow().strftime('%Y_%m_%d_%H_%M_%S'))
```

Comment:
Please share more details on why we need this change. I have this question because we display two timestamps in each line. The first timestamp is this one which we are trying to change to utcnow(). The second time stamp is put I believe by **platform code** when system goes down. 

**2025_07_28_13_34_54**  reboot      **Mon Jul 28 01:32:25 PM IDT** 2025  admin   N/A 

So if we change at one place we should change at both places? 

Ideally I feel if the system tz is configured as UTC, both timestamps will be in UTC.

Reply:
> So if we change at one place we should change at both places?
OK, will update it.
Previously I though maybe can use original timestamp in 2nd place, because it can show original time zone when system is running, so we can have more info.
1st place, we have to align to same timezone like UTC, because it is used for sort.

Reply:
Hello @judyjoseph 

I haven't tested those PRs together with mine, but do have test those 2 PRs in local, seems this bug not fixed my issue. so I tried to update these timestamps to UTC in reboot-casue, to fix reboot part issue.